.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: Build
```

Comment:
please, provide more specific name for the action, like "Build VSIX" or "Build VS Code Extension"